rust_src/src/remacs_sys.rs

```diff
@@ -1748,7 +1758,7 @@ pub mod font {
 
 #[cfg(test)]
 macro_rules! offset_of {
-    ($ty: ty, $field: ident) => {
+    ($ty:ty, $field:ident) => {
```

Comment:
You are failing rustfmt because you likely have a newer rustfmt installed. This section should not have changed.

Reply:
So many "standards" to choose from! How do I get the right rustmt installed? 

```
$ rustup component add rustfmt-preview 
info: component 'rustfmt-preview' for target 'x86_64-unknown-linux-gnu' is up to date
```

Ok good. Then I try: 
```
$ (cd rust_src && cargo fmt -- --write-mode overwrite) 
error: no such subcommand: `fmt`
```
I guess I have the wrong version of cargo? If that is the case, how do I get the right one so I can run with the right version of rustfmt? Thanks

Reply:
First section in README says how to get proper version of Rust for this project. I guess you didn't run `rustup` from the Remacs root.

To format all the files, run `make rustfmt`.